src/api/app/views/webui2/webui/packages/job_history/index.html.haml

```diff
@@ -0,0 +1,48 @@
+- @pagetitle = "Job history of #{@project} / #{@package_name}"
+- @metarobots = 'index,nofollow'
+
+.card
+  = render partial: 'webui/package/tabs', locals: { project: @project, package: @package }
+
+  .card-body
+    %h3= @pagetitle
+    %h6.subtitle
+      Repository / Architecture: #{params[:repository]} / #{params[:arch]}
+
+    %table.responsive.table.table-striped.table-bordered#jobhistory-table{ width: '100%' }
+      %thead
+        %th Revision
+        %th Time (not formatted)
+        %th Time
+        %th Reason
+        %th Status Code
+        %th Build Time (not formatted)
+        %th Build Time
+        %th Worker
+        %th
+      %tbody
+        - @jobshistory.each do |jobhistory|
+          %tr
+            %td
+              = jobhistory.revision
+            %td
+              = jobhistory.ready_time
+            %td
+              = time_tag(Time.at(jobhistory.ready_time))
+            %td
+              - if jobhistory.reason == 'source change'
+                = link_to(jobhistory.reason, package_rdiff_path(project: @project.name,
+                  package: @package, orev: jobhistory.prev_srcmd5, rev: jobhistory.srcmd5))
+              - else
+                = jobhistory.reason
+            %td{ class: "status_#{jobhistory.code}" }
+              = jobhistory.code
+            %td
+              = jobhistory.total_time
+            %td
+              = humanize_time(jobhistory.total_time)
+            %td
+              = jobhistory.worker_id
+            %td
+              = link_to_package_from_job_history(@project, @package,
```

Comment:
I'm not sure we should keep using this helper. It's only used once and it's pretty simple. It's also using a [sprite_tag](https://github.com/openSUSE/open-build-service/blob/f66cdf332289be2e54186eaff5752b19b29716d5/src/api/app/helpers/webui/packages/job_history_helper.rb#L9) which we don't want to use in the Bootstrap UI.

It's breaking the UI right now because we don't see anything.
![review-job-history](https://user-images.githubusercontent.com/1102934/46106714-c27f8080-c1d9-11e8-9bb5-51ddc1d69b8f.png)